### PR TITLE
Alert Popup feature imported successfully in contact page

### PR DESCRIPTION
**Pull request by ROHAN842**
Fixes Issue #725 
I successfully added an alert which will pops up and will tell the user that message was delivered successfully with a green tick along with I also added a feature without back end with which user would be sent to a fresh new page on clicking Send button where the message is displayed as  "Thank you We received your submission , Return ".

https://user-images.githubusercontent.com/61961107/122445187-ed7c1500-cfbe-11eb-89e3-ef63e3d3ff12.mp4

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/40017559/122457702-8e250180-cfcc-11eb-80b1-dafa915b2146.mp4

It should validate all fields before sending them to the next page. Still, it show's congrats so check these and add them.

File: assets/js/app.js [hunk 1]
```diff
@@ -199,50 +199,55 @@ let footer = $(`
...
             </form>
           </div>
         </div>
       </div>
     </div>
   </div>
 </footer>
+<script type="text/javascript">
+        function JSalert() {
+            swal("Congrats!", "Message was delivered successfully!", "success");
```
Comment (smaranjitghose) [inline assets/js/app.js:216]: Make this as:

```
Thank you for the response. John will get in touch soon!
```

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: assets/js/app.js (hunk 1 above)
Comment (ROHAN842) [reply assets/js/app.js:216]: OK I will resolve this

**Comment by ROHAN842** [comment]
Hey @anushbhatia as you have already merged my updated PR so I request you to kindly close this PR

**Pull request closed without merging**